tests/jax/models/Roberta/test_roberta.py

```diff
@@ -0,0 +1,71 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Dict, Sequence
+
+import jax
+import pytest
+from flax import linen as nn
+from infra import ModelTester, RunMode
+from transformers import AutoTokenizer, FlaxRobertaForMaskedLM
+
+MODEL_PATH = "FacebookAI/roberta-base"
```

Comment:
Seems like Roberta also has base, large, etc versions. Don't do it in this PR, but maybe in some of the future PRs you can cover other versions too, similar as you did for Albert.